pkg/operation/botanist/component/logging/fluentoperator/fluent_operator.go

```diff
@@ -0,0 +1,501 @@
+				},
+				{
+					APIGroups: []string{"fluentbit.fluent.io"},
+					Resources: []string{"fluentbits", "clusterfluentbitconfigs", "clusterfilters", "clusterinputs", "clusteroutputs", "clusterparsers"},
+					Verbs:     []string{"create", "delete", "get", "list", "patch", "update", "watch"},
+				},
+				{
+					APIGroups: []string{"fluentd.fluent.io"},
+					Resources: []string{"fluentds", "fluentds/status", "clusterfluentdconfigs", "clusterfluentdconfigs/status", "fluentdconfigs", "fluentdconfigs/status", "clusterfilters", "filters", "clusteroutputs", "outputs"},
+					Verbs:     []string{"create", "delete", "get", "list", "patch", "update", "watch"},
+				},
+				{
+					APIGroups: []string{"rbac.authorization.k8s.io"},
+					Resources: []string{"clusterrolebindings", "clusterroles"},
+					Verbs:     []string{"create"},
+				},
+				{
+					APIGroups: []string{"extensions.gardener.cloud"},
+					Resources: []string{"clusters"},
+					Verbs:     []string{"get", "list", "watch"},
+				},
+			},
+		}
+		clusterRoleBinding = &rbacv1.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   name,
+				Labels: f.getLabels(),
+			},
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: rbacv1.GroupName,
+				Kind:     "ClusterRole",
+				Name:     clusterRole.Name,
+			},
+			Subjects: []rbacv1.Subject{{
+				Kind:      rbacv1.ServiceAccountKind,
+				Name:      serviceAccount.Name,
+				Namespace: serviceAccount.Namespace,
+			}},
+		}
+		deployment = &appsv1.Deployment{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: f.namespace,
+				Labels:    f.getLabels(),
+			},
+			Spec: appsv1.DeploymentSpec{
+				RevisionHistoryLimit: pointer.Int32(1),
+				Replicas:             pointer.Int32(1),
+				Selector: &metav1.LabelSelector{
+					MatchLabels: f.getLabels(),
+				},
+				Template: corev1.PodTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: f.getLabels(),
+					},
+					Spec: corev1.PodSpec{
+						ServiceAccountName: name,
+						PriorityClassName:  f.values.OperatorPriorityClassName,
+						Containers: []corev1.Container{
+							{
+								Name:            name,
+								Image:           f.values.OperatorImage,
+								ImagePullPolicy: corev1.PullIfNotPresent,
+								Env:             f.computeEnv(),
+								Resources: corev1.ResourceRequirements{
+									Requests: corev1.ResourceList{
+										corev1.ResourceCPU:    resource.MustParse("20m"),
+										corev1.ResourceMemory: resource.MustParse("50Mi"),
+									},
+								},
+								VolumeMounts: f.computeVolumes(),
+							},
+						},
+						Volumes: []corev1.Volume{
+							{
+								Name: "env",
+								VolumeSource: corev1.VolumeSource{
+									EmptyDir: &corev1.EmptyDirVolumeSource{},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+		vpaUpdateMode = vpaautoscalingv1.UpdateModeAuto
+		vpa           = &vpaautoscalingv1.VerticalPodAutoscaler{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name + "-vpa",
```

Comment:
I know that most of the current VPA objects have `-vpa` suffix, but it is not needed actually. We don't suffix in the same way Services, Deployments, etc.

```suggestion
				Name:      name,
```